docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -265,7 +265,7 @@ type of check. You can find the possible properties for all the checks below.
 
 ==== _Datafeed is not started_ 
 
-`context.message`::
+`context.message` ^*^::
```

Comment:
For the Anomaly detection job health rule, `context.results` is also available for all checks, except the `Errors in job message`